content-src/asrouter/components/ModalOverlay/ModalOverlay.jsx

```diff
@@ -51,7 +51,6 @@ export class ModalOverlayWrapper extends React.PureComponent {
       <React.Fragment>
         <div
           className="modalOverlayOuter active"
-          onClick={props.onClose}
```

Comment:
Can you add a comment that mentions "The intended behaviour is to listen for an escape key but not for a click; see Bug 1582242"

Reply:
done